desc/io/input_reader.py

```diff
@@ -834,7 +834,32 @@ def parse_vmec_inputs(vmec_fname, threshold=0):  # noqa: C901 - FIXME: simplify
         pres_scale = 1.0
         curr_tor = None
 
-        for line in vmec_file:
+        # Loop which makes multi-line inputs a single line
+        vmeclines_no_multiline = []
+
+        for line in vmec_file.readlines():
+            comment = line.find("!")
+            line = (line.strip() + " ")[0:comment].strip()
+            # ignore blank lines or leftover comment !'s
+            if line.isspace() or line == "" or line == r"!":
+                continue
+            if line.find("&INDATA") != -1:
+                continue
+            elif line.find("&") != -1:  # only care about the INDATA section
+                break
+            if line.find("=") != -1:
+                vmeclines_no_multiline.append(line)
+            else:  # is a multi-line input,append the line to the previous
+                vmeclines_no_multiline[-1] += " " + line
+
+        # Some bool flags to ensure that if a line is duplicate in
+        # the VMEC file, we do not add it twice to the inputs
+        RAXIS_already_read = False
```

Comment:
why just these keys? Are there other things that might be written twice? And do we want to throw an error in that case? IE, how do we know which one is the "right" axis?